design/ceph/ceph-nfs-ganesha.md

```diff
@@ -115,6 +115,32 @@ spec:
     #    memory: "1024Mi"
     # the priority class to set to influence the scheduler's pod preemption
     priorityClassName:
+
+  security:
+    systemSecurityServicesDaemon:
+      runInSidecar:
+        image: registry.access.redhat.com/rhel7/sssd:latest
+        config:
+          subPath: sssd.conf
+          volume:
+            # any standard Kubernetes volume definition
+            # e.g., a configmap
+            name: sssd-config
+            configMap:
+              name: sssd-config
+              defaultMode: 0600 # required
+            # # or, e.g., a hostPath
+            # name: sssd-config
+            # hostPath:
+            #   path: /etc/sssd/conf.d
+            #   type: Directory
```

Comment:
Instead of this, I'm wondering if we should just enforce that users should use a configmap. We can then also make it a requirement that the conf file in the configmap be mountable as `sssd.conf`. 

If we are adding an option to use the host's SSSD later, that will take care of the case of using `sssd.conf` from the host.